discretize/TreeMesh.py

```diff
@@ -1291,14 +1360,15 @@ def _deflationMatrix(self, location, withHanging=True, asOnes=False):
             args['Ex'] = (self._facesY, self._hangingFy, self._fy2i)
             args['Ey'] = (self._facesX, self._hangingFx, self._fx2i)
         if location in ['F', 'E']:
-            Rlist = [self._deflationMatrix(location + subLoc, withHanging=withHanging, asOnes=asOnes) for subLoc in ['x','y','z'][:self.dim]]
+            Rlist = [self._deflationMatrix(location + subLoc, withHanging=withHanging, asOnes=asOnes, large=large) for subLoc in ['x', 'y', 'z'][:self.dim]]
```

Comment:
Where is `large` coming from?

Reply:
Was playing around with a deflation that preserved the hanging faces. Has been removed.